### PR TITLE
Update for 1.11.0 release candidate

### PR DESCRIPTION
**Pull request by michalbiesek**
TODO:

- [x] merge hmat-devel branch

---
This change is [Reviewable](https://reviewable.io/reviews/memkind/memkind/496)

### CODE REVIEW

**Review by bratpiorka — COMMENTED** [review]
> Reviewed 3 of 3 files at r1.
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/496)* status: all files reviewed, 1 unresolved discussion (waiting on @michalbiesek)

___
*[VERSION, line 1 at r1](https://reviewable.io/reviews/memkind/memkind/496#-MS2RraV3EUctnINLkFi:-MS2RraWE63HtA6dbBUq:b-bmaf4b) ([raw file](https://github.com/memkind/memkind/blob/7e033412e3f529f0e914ff931f0a15a8d134a232/VERSION#L1)):*
> ```
> 1.11.0rc1
> ```

nit: 1.11.0-rc1 ?

**Comment by michalbiesek** [comment]
> > Reviewed 3 of 3 files at r1.
> > _[Reviewable](https://reviewable.io/reviews/memkind/memkind/496)_ status: all files reviewed, 1 unresolved discussion (waiting on @michalbiesek)
> 
> _[VERSION, line 1 at r1](https://reviewable.io/reviews/memkind/memkind/496#-MS2RraV3EUctnINLkFi:-MS2RraWE63HtA6dbBUq:b-bmaf4b) ([raw file](https://github.com/memkind/memkind/blob/7e033412e3f529f0e914ff931f0a15a8d134a232/VERSION#L1)):_
> 
> > ```
> > 1.11.0rc1
> > ```
> 
> nit: 1.11.0-rc1 ?

I prefer without a hyphen, see https://github.com/memkind/memkind/commit/c62bb9cec08e80a979bb1126ad7a8bcd6329a5c4 or https://github.com/memkind/memkind/commit/b59a031aabd5f5610ccbb7d47895ce1929ebcfc7

**Review by bratpiorka — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/496)* status: :shipit: complete! all files reviewed, all discussions resolved

___
*[VERSION, line 1 at r1](https://reviewable.io/reviews/memkind/memkind/496#-MS2RraV3EUctnINLkFi:-MS2TUx_860twhvgCBU0:b-4ey8vg) ([raw file](https://github.com/memkind/memkind/blob/7e033412e3f529f0e914ff931f0a15a8d134a232/VERSION#L1)):*
<details><summary><i>Previously, michalbiesek wrote…</i></summary><blockquote>

I prefer without a hyphen, see https://github.com/memkind/memkind/commit/c62bb9cec08e80a979bb1126ad7a8bcd6329a5c4 or https://github.com/memkind/memkind/commit/b59a031aabd5f5610ccbb7d47895ce1929ebcfc7 
</blockquote></details>

ok I saw that this is different in Changelog, but keep it as it is to be consistent with past commits

**Review by PatKamin — COMMENTED** [review]
> Reviewed 3 of 3 files at r1.
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/496)* status: all files reviewed, 1 unresolved discussion (waiting on @michalbiesek)

___
*[ChangeLog, line 7 at r1](https://reviewable.io/reviews/memkind/memkind/496#-MS2gVW6EAesMzHrquiP:-MS2gVW6EAesMzHrquiQ:bqw2nxv) ([raw file](https://github.com/memkind/memkind/blob/7e033412e3f529f0e914ff931f0a15a8d134a232/ChangeLog#L7)):*
> ```
> - Provided memory property abstraction (MEMKIND_HIGHEST_CAPACITY*,
>   MEMKIND_LOWEST_LATENCY_LOCAL*, MEMKIND_HIGHEST_BANDWIDTH_LOCAL*)
> - Extended support for MEMKIND_HBW*, for platforms with full HMAT support (kernel+hardware)
> ```

MEMKIND_HBW* to platforms

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,11 @@
+* Fri Jan 29 2021 Michal Biesek <michal.biesek@intel.com> v1.11.0-rc1
+- Provided function to verify if path supports DAX - memkind_check_dax_path()
+- Provided an interleave variant for MEMKIND_DAX_KMEM (MEMKIND_DAX_KMEM_INTERLEAVE)
```
Comment (jschmieg) [inline ChangeLog:3]: "node interleave" or "NUMA node interleave"

**Review by PatKamin — COMMENTED** [review]
> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/496)* status: all files reviewed, 1 unresolved discussion (waiting on @michalbiesek)

**Pull request merged**